builtins/default.nix

```diff
@@ -7,54 +7,57 @@
 
 rec
 {
-    toTOML = import ./to-toml.nix { inherit lib; };
-    writeTOML = name: attrs: writeText name (toTOML attrs);
+  toTOML = import ./to-toml.nix { inherit lib; };
+  writeTOML = name: attrs: writeText name (toTOML attrs);
 
-    readTOML = usePure: f:
-      if usePure then
-        builtins.fromTOML (builtins.readFile f)
-      else
-        builtins.fromJSON (builtins.readFile (
+  readTOML = usePure: f:
+    if usePure then
+      builtins.fromTOML (builtins.readFile f)
+    else
+      builtins.fromJSON (
+        builtins.readFile (
           runCommand "from-toml"
-          { buildInputs = [ remarshal ];
-            allowSubstitutes = false;
-            preferLocalBuild = true;
-          }
-          ''
-            echo "$from_toml_in" > in.toml
-            remarshal \
-              -if toml \
-              -i ${f} \
-              -of json \
-              -o $out
-          ''));
+            {
+              buildInputs = [ remarshal ];
+              allowSubstitutes = false;
+              preferLocalBuild = true;
+            }
+            ''
+              echo "$from_toml_in" > in.toml
+              remarshal \
+                -if toml \
+                -i ${f} \
+                -of json \
+                -o $out
+            ''
+        )
+      );
 
-    writeJSON = name: attrs: writeText name
-      (builtins.toJSON attrs);
+  writeJSON = name: attrs: writeText name
+    (builtins.toJSON attrs);
 
-    # Returns `true` if `path` exists.
-    # TODO: use `builtins.pathExists` once
-    # https://github.com/NixOS/nix/pull/3012 has landed and is generally
-    # available
-    pathExists = path:
-      let
-        all = lib.all (x: x);
-        isOk = part:
-          let
-            dir = builtins.dirOf part;
-            basename = builtins.unsafeDiscardStringContext (builtins.baseNameOf part);
-            dirContent = builtins.readDir dir;
-          in
-          builtins.hasAttr basename dirContent &&
-          # XXX: this may not work if the directory is a symlink
+  # Returns `true` if `path` exists.
+  # TODO: use `builtins.pathExists` once
+  # https://github.com/NixOS/nix/pull/3012 has landed and is generally
+  # available
+  pathExists = path:
+    let
+      all = lib.all (x: x);
+      isOk = part:
+        let
+          dir = builtins.dirOf part;
+          basename = builtins.unsafeDiscardStringContext (builtins.baseNameOf part);
+          dirContent = builtins.readDir dir;
+        in
+          builtins.hasAttr basename dirContent && # XXX: this may not work if the directory is a symlink
```

Comment:
@zimbatm looks like this guy got moved! Issue when formatting infix operators?

Reply:
hmm maybe. I'll just fix it manually for now

Reply:
moved the code around a bit